server/serverbase.js

```diff
@@ -104,10 +104,18 @@ exports.initialize2 = function(app) {
       }
     }
   }
+  function loadThumb() {
```

Comment:
Are we gonna have thumbs for folders? If we are only to show the folder icon for folders, then calling `loadThumb` with a directory name should just return 404.